Feature/user update test

lexville opened the pull request.

lexville committed:
Make Update Profile
[Finishes #109263336]
The user can now update their profile

lexville committed:
Fix StyleCi errors
References [#109263336]
The changes were necessitated by styleci

lexville committed:
Fix StyleCI errors
Reference [#109263336]
Changes made due to styleci errors

lexville committed:
Fix Social Registration Request style errors
References [#109263336]
Changes made as a result of styleci failure

Commit:
fix tests

lexville committed:
Fix dashboard controller
Reference [#109263336]
styleci errors for dashboard controller fixed

Commit:
Merge branch 'feature/updateProfile' of github.com:andela/resly into feature/updateProfile

lexville committed:
Merge branch 'feature/updateProfile' of https://github.com/andela/resly into feature/updateProfile

* 'feature/updateProfile' of https://github.com/andela/resly:
  fix tests

lexville committed:
Fix styleci errors
Reference [#109263336]
The Database seeder styleci errors are now fixed

Commit:
fix tests

Commit:
Merge branch 'feature/updateProfile' of github.com:andela/resly into feature/updateProfile

lexville committed:
Merge branch 'develop' into feature/updateProfile
Reference [#109263336]
Fix merge conflicts
* develop:
  ignore .DS_Store files
  remove system artifacts
  remove abandoned auth controllers
  remove google auth keys from .env.example
  Fixing more style issues
  Fix style issues
  fix diner integrity error int diner test
  redirect to dashboard
  fix login tests

lexville committed:
Fix merge conflict Reference[#109263336] Fixed merge conflicts

lexville committed:
Fix styleci errors References[#109263336] Fixed styleci errors in RestaurantController

lexville committed:
Fix styleci errors References[#109263336] Fixed styleci errors in DinerLoginTest

lexville committed:
Make UpdateProfileTest
[Finishes #109702174]
User update profile tests are now available

andela-anandaa commented:
@andela-akangethe - The tests look fine, but I'd asked that you squash the repetitive commits and force push -- commits `7a505b3` to `6ba7348` could just be "Fix Merge conflicts".

lexville committed:
Fix merge conflict
Merge branch 'develop' into feature/UserUpdateTest

* develop:
  add restaurant details
  Deleted multi-auth related files
  reduce footer further reduce top padding for footer, remove paragraph bottom margin and font-size.
  remove conflict markers
  reduce footer top margin [finishes #109702064]
